modules/fts/pages/fts-searching-with-the-rest-api.adoc

```diff
@@ -100,6 +100,37 @@ http://localhost:18094/api/index/travel-sample-index/query \
 For additional information on secure communication with Couchbase Server, see xref:manage:manage-security/manage-connections-and-disks.adoc[Manage Connections and Disks].
 
 
+[#searching-with-query-timeouts]
+== Searching with timeouts
+
+Once the the index becomes really large, it is possible that certain search requests may take
+longer time to finish the execution.Such slow responses would affect the query latencies as well as the resource consumption on the server side.
+
+For avoiding such unplesant situation, users can pass a search "timeout" value in the request.
```

Comment:
Typo: unpleasant* situations*

Feels a bit harsh though, perhaps we should drop the entire line..